label_studio/ml/serializers.py

```diff
@@ -5,6 +5,7 @@
 from ml.models import MLBackend
 from core.utils.io import validate_upload_url
 
+
```

Comment:
this blank line is extraneous, would favor removing to keep blame clean

Reply:
the same as previous comment..